nexus-sdk/guides/setup.md

```diff
@@ -74,13 +76,28 @@ credentials. To request access, please submit your details using the form\
 provided in the navigation bar.
 {% endhint %}
 
-Once you receive your credentials, configure your Nexus CLI to connect to the\
+Once you receive your credentials, first set them as environment variables in your terminal.\
+Replace `my-username` and `my-password` with your actual credentials:
+
+```bash
+export NEXUS_USERNAME="my-username"
+export NEXUS_PASSWORD="my-password"
+```
+
+Clone the nexus-sdk repository
+
+```code
+git clone --depth 1 --branch v0.1.0 https://github.com/Talus-Network/nexus-sdk.git
+cd nexus-sdk
+```
```

Comment:
This is unnecessary, the users shouldn't need to clone the repo since they can install the CLI using the provided installers (Homebrew, AUR, and in last instance cargo).